src/timepicker/example/index.ts

```diff
@@ -0,0 +1,172 @@
+import { DNode } from '@dojo/widget-core/interfaces';
+import { ProjectorMixin } from '@dojo/widget-core/mixins/Projector';
+import { StatefulMixin } from '@dojo/widget-core/mixins/Stateful';
+import { v, w } from '@dojo/widget-core/d';
+import { WidgetBase } from '@dojo/widget-core/WidgetBase';
+import { WidgetProperties } from '@dojo/widget-core/interfaces';
+import TimePicker, { TimeUnits } from '../TimePicker';
+import dojoTheme from '../../themes/dojo/theme';
+
+export class App extends StatefulMixin(WidgetBase)<WidgetProperties> {
+	private _theme: {};
+
+	onRequestOptions(value: string, options: TimeUnits[]) {
+		this.setState({ options });
+	}
+
+	themeChange(event: Event) {
+		const checked = (<HTMLInputElement> event.target).checked;
+		this._theme = checked ? dojoTheme : {};
+		this.invalidate();
+	}
+
+	render(): DNode {
+		return v('div', [
+			v('h1', [ 'TimePicker Examples' ]),
+			v('label', [
+				'Use Dojo Theme ',
+				v('input', {
+					type: 'checkbox',
+					onchange: this.themeChange
+				})
+			]),
+			v('h3', [ 'Filter options on input' ]),
+			w(TimePicker, {
+				inputProperties: {
+					placeholder: 'Enter a value'
```

Comment:
Since we're going the `inputProperties` route for a11y, could we add `describedBy` to all the `inputProperties` examples to model good a11y behavior? E.g.
```js
v('p', {
   id: 'foo',
   classes: this.classes().fixed(baseCss.visuallyHidden)
}, [ 'Accepts 24-hour time with a leading zero, rounded to the nearest half hour' ])
...
inputProperties: {
   placeholder: 'Do things',
   describedBy: 'foo'
}
```
I just want to make sure we model doing this because otherwise there's no indication of what the time format should be if you can't see the combobox options.